GameOfLife/Displaying/DisplayGameField.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace GameOfLife
+{
+    public class DisplayGameField
+    {
+
+        public void PrintArray(int[,] gameField)
+        {
+            for (int i = 0; i < gameField.GetLength(0); i++)
+            {
+                for (int j = 0; j < gameField.GetLength(1); j++)
+                {
+                    Console.Write(gameField[i, j] + " ");
+
+                }
+                Console.WriteLine();
+
+            }
+        }
+        /// <summary>
+        /// Prints generated starting field base on CellPupulator class
+        /// </summary>
+        /// <param name="gameFieldArray"></param>
+        public void PrintStartingArray(int[,] gameFieldArray)
+        {
+            Console.WriteLine("Starting field");
+            Console.WriteLine();
+            DisplayGameField gameIterator = new DisplayGameField();
```

Comment:
You are already inside the DisplayGameField class, you can access the PrintArray method without creating a new object.

Reply:
Deleted unnecessary code